NuGet.Config

```diff
@@ -1,10 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <packageSources>
         <clear />
         <!-- Dependencies that we can turn on to force override for testing purposes before uploading. -->
         <!--<add key="Static Package Dependencies" value="dep\packages" />-->
         <add key="TerminalDependencies" value="https://pkgs.dev.azure.com/ms/terminal/_packaging/TerminalDependencies/nuget/v3/index.json" />
+        <!-- !!! THIS NEEDS TO BE REMOVED BEFORE MERGE. The Microsoft.VisualStudio.Setup.Configuration.Native package needs to be added to the upstream source !!! -->
+    	<add key="nuget.org" value="https://api.nuget.org/v3/index.json"/>

```

Comment:
Nit: tabbing off.

Reply:
I'm not worried about it, becuase it needs to be removed before this can merge.

Reply:
@zadjii-msft , @DHowett How do I get Microsoft.VisualStudio.Setup.Configuration.Native served from your single nuget source?

Reply:
I'm gonna also /cc @miniksa since he's most familiar with the CI stuff and can probably get to this before me

Reply:
During conflict resolution, you can finally back this one on out :smile: